apps/ledger-live-mobile/README.md

```diff
@@ -59,10 +59,14 @@ pnpm dev:llm -- --reset-cache
 
 ### `pnpm mobile ios`
 
+/!\ Before running this the first time you will need to do a `pnpm build:llm:deps` then you will be able to run it on ios
+
 or `open ios/ledgerlivemobile.xcworkspace` in XCode
 
 ### `pnpm mobile android`
 
+/!\ Before running this the first time you will need to do a `pnpm build:llm:deps` then you will be able to run it on ios
```

Comment:
It says `*ios*` but it is under the `pnpm mobile android` section.

Also I'm not sure this is correct for both android and ios.

`pnpm build:llm:deps` is only required to use metro, which is `pnpm mobile start`, not `pnpm mobile ios` / `pnpm mobile android`.

It is also useless when using `pnpm dev:llm` (which should be the default anyway).

Reply:
I'll fix the wording and stuff. 

I got one issue about an external dev that wasn't able to run ios before doing a build:lib . 
<img width="835" alt="Screenshot 2022-09-22 at 18 47 05" src="https://user-images.githubusercontent.com/33897120/191805583-da57e820-f25e-4680-8178-8844b5ab789c.png">

I 'll try to run it again with pnpm:clean and tell you about the result of it

Reply:
@elbywan if you just clone the repo, you execute `pnpm install`. Then if you want to run `pnpm mobile ios`, the process will fail as some fonts from ui-react are required, and they won't be there.

Reply:
I'm doing `pnpm clean`, `pnpm i`, `pnpm mobile ios` quite often so I'm not sure this is correct.

Reply:
Please, try to clone the repo from fresh, and do `pnpm I` and  `pnpm mobile ios`. I don't have the output anymore to put it here.

Reply:
Yeah, it's seems that he did not run mobile start first, I'll just left a note about this. Because it couldn't be always an issue.

Reply:
@emmanuelm41 My bad, you're right fonts are an issue for ios 🙇. 
On the other hand, android build fine on my end.


> Yeah, it's seems that if we do not run pnpm mobile start first it will failed, I'll just left a note about this.

Yup this is better IMO this way. The only thing is we should not mention `pnpm mobile start` at all, we should tell people to use `pnpm dev:llm` which will take care of building all the dependencies.

Using `pnpm dev:llm` then `pnpm mobile ios` / `pnpm mobile android` should be the way to go and work for everyone.

Reply:
Agree 💯